packages/webapp/src/components/Form/InputBase/index.tsx

```diff
@@ -71,18 +71,18 @@ const InputBase = forwardRef<HTMLInputElement, InputBaseProps>((props, ref) => {
             toolTipContent={toolTipContent}
           />
         )}
-        <InputBaseField
-          {...inputProps}
-          leftSection={leftSection}
-          mainSection={mainSection}
-          rightSection={rightSection}
-          isError={!!error}
-          resetIcon={
-            !!error && showResetIcon ? <Cross isClickable onClick={onResetIconClick} /> : undefined
-          }
-          ref={ref}
-        />
       </label>
```

Comment:
The `<label>` tag here is meant to wrap the inputField for accessibility so the Input can have an associated label because InputBaseLabel doesn't use the `label` tag. Does it need to be removed?

Reply:
Sorry 😅, I reverted it.
`<label>` closes ReactSelect dropdown right after it opens, but I wrapped ReactSelect and added `preventDefault()` to prevent it.